http/annotated-routes.md

```diff
@@ -47,6 +47,7 @@ methods | array/string | HTTP methods. Defaults to all methods.
 defaults | array | Default values for route pattern.
 group | string | Route group, defaults to `default`.
 middleware | array | Route specific middleware class names.
+priority | int | Position in a routes list. Higher priority routes are sorted before lower ones. Helps to solve situations when one request matches two routes. Defaults to 0.
```

Comment:
Same. While there is no versioning of the documentation, the description of new functions should preferably include the version with which the feature is available.